__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require('../app');
+const runSeed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const request = require('supertest');
+beforeEach(() => runSeed(testData));
+describe('GET non-existent endpoint', () => {
+    test("returns 404", () => {
+        return request(app)
+        .get('/api/cagetories')
+        .expect(404);
+    });
+})
+describe('GET /api/categories', () => {
+    test("returns status code 200", () => {
+        return request(app)
+        .get('/api/categories')
+        .expect(200);
+    });
+    test("returns array of objects", () => {
+        return request(app)
+        .get('/api/categories')
+        .then((res) => {
+            expect(Array.isArray(res.body)).toBe(true);
+        })
+    })
+    test("returns correct response", () => {
```

Comment:
this is a brittle test, it's the equivalent of saying you expect an object to equal another object which isn't robust enough. It works for now with categories because there are no IDs in the data, but won't hold up for future tests. We'd like to see jest matchers .toMatchObject or .objectContaining here please